server/database/schema.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+mongoose.connect(process.env.MONGO_URI, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true
+})
+  .catch(err => console.error(err));
+
+const userSchema = new mongoose.Schema({
+  userId: Number,
+  name: String,
+  joinDate: String,
```

Comment:
Something I learned is that mongoose has a 'Date' type, which is aligned with the javascript date objects: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date
It makes handling dates and manipulating them a lot easier! Even if you store the date as a string, you can do a new Date(strDate) in your code later to convert it back.

Reply:
Nice! I'll have to take a look. Thanks for the suggestion.